hlx_statics/blocks/columns/columns.css

```
@@ -196,8 +208,10 @@ main div.columns-wrapper div.columns.center .button-container {
 
 @media screen and (max-width: 1280px) {
     main div.columns-wrapper  div.columns div.second-column {
-        width: 100%;
-        margin: 40px 0;
+        width: 75vw;
```

I think % is better when we have nested divs and we want it to take the parent’s width. vw and % are equivalent when the parent div === viewport

```suggestion
        width: 75%;
```